tests/cypress/support/commands.js

```diff
@@ -39,7 +39,18 @@ Cypress.Commands.add(
     var user = OPTIONS_HUB_USER || Cypress.env('OPTIONS_HUB_USER')
     var password = OPTIONS_HUB_PASSWORD || Cypress.env('OPTIONS_HUB_PASSWORD')
     var idp = OPTIONS_HUB_OC_IDP || Cypress.env('OPTIONS_HUB_OC_IDP')
-    cy.visit('/search')
+    cy.visit('/search', { failOnStatusCode: false })
+    cy.url().then((url) => {
+      if (!url.includes('oauth-openshift')) {
+        // check for and handle provider button
```

Comment:
@KevinFCormier I'm curious, is there another way to determine by the url if we are on a page with the `Log in with Openshift button`? In `2.5`, we check a similar condition; however, if the URL doesn't contain `oauth-openshift`, we assume that were already logged in: https://github.com/stolostron/search-e2e-test/blob/main/tests/cypress/support/commands.js#L45-L64.

I'm worried if this will cause errors with that login flow also since we intend on back porting some of those changes within `2.4`.

Reply:
Not from the URL - because it comes from management-ingress, it is the URL we are trying to test:
<img width="872" alt="image" src="https://user-images.githubusercontent.com/42188127/163269283-e27a52b8-af58-4b9d-a314-74b37d116b3b.png">

This is why I also checked for lack of the element with ID `header` before trying to click the provider button, based on the code I saw here and in 2.5. (See PR: https://github.com/stolostron/search-e2e-test/pull/261)

However, I am confused about how this is working, because I don't see an element with ID `header` in 2.4. I do see a `<header>` element though.